xdem/coreg/biascorr.py

```diff
@@ -118,72 +126,14 @@ def __init__(
                 "fit_optimizer": fit_optimizer,
                 "bin_sizes": bin_sizes,
                 "bin_statistic": bin_statistic,
+                "bias_var_names": list_bias_var_names,
             }
             super().__init__(meta=meta_bin_and_fit)  # type: ignore
 
         # Update attributes
         self._fit_or_bin = fit_or_bin
         self._is_affine = False
-
```

Comment:
Used to have these two here to override `Coreg.fit()` and `.apply()` (and not only the `._fit_func()`, `._apply_func()`), but now that `bias_vars` has been moved to `Coreg` directly, there is no need for them anymore.